cluster/src/main/java/com/hartwig/pipeline/Arguments.java

```diff
@@ -11,6 +11,8 @@
 
     String EMPTY = "";
 
+    Optional<String> runFrom();
```

Comment:
Feels to me like `run` is a bit overloaded already. Could we use another verb? Like `begin` or `start` or `restart`?